### DISCUSSION TITLE
Support secret in docker compose

### DISCUSSION THREAD

**Discussion by shuther** [langfuse/langfuse#902] [Ideas]
Would it be possible to support extra environment variables such as NEXTAUTH_SECRET_FILE, so we can make use of secrets within docker compose? A good example is from [example from dolibarr](https://github.com/tuxgasy/docker-dolibarr/blob/master/docker-compose.yml) that is showing how it woks already with postgresql.

**Comment by maxdeichmann** [comment]
Yes, this would be a great addition. Would you be interested in contributing to that?
